library/std/src/intrinsic.qs

```diff
@@ -368,17 +368,12 @@ namespace Microsoft.Quantum.Intrinsic {
     /// The output result is given by the distribution:
     /// $$
     /// \begin{align}
-    ///     \Pr(\texttt{Zero} | \ket{\psi}) =
-    ///         \frac12 \braket{
-    ///             \psi \mid|
-    ///             \left(
-    ///                 \boldone + P_0 \otimes P_1 \otimes \cdots \otimes P_{N-1}
-    ///             \right) \mid|
-    ///             \psi
-    ///         },
+    ///     \langle\psi|
```

Comment:
Do our docs understand \bra and \ket macros instead of \langle and \rangle? I know the katas do and I'm trying to use them for readability, but I haven't had the chance to check the docs

Reply:
That fixed version I mentioned in a different comment uses bra and ket.

Reply:
Yes, the updated description now uses \bra and \ket.